aten/src/ATen/core/PythonFallbackKernel.cpp

```diff
@@ -0,0 +1,35 @@
+#include <torch/library.h>
+#include <ATen/core/dispatch/Dispatcher.h>
+
+namespace {
+
+void pythonFallback(const c10::OperatorHandle& op, torch::jit::Stack* stack) {
+  const auto& schema = op.schema();
+  const auto num_arguments = schema.arguments().size();
+  for (const auto& ivalue : torch::jit::last(*stack, num_arguments)) {
+    if (ivalue.isTensor()) {
+      auto* interpreter = ivalue.unsafeToTensorImpl()->pyobj_interpreter();
+      if (interpreter) {
+        interpreter->dispatch(op, stack);
+        return;
+      }
+    } else if (ivalue.isList()) {
+      for (const auto& nv : ivalue.toListRef()) {
+        if (nv.isTensor()) {
```

Comment:
not a big deal, but does this code imply that your `List` ivalue might have mixed types, some of which are Tensors and some that aren't? If not, maybe `isTensorList()` is a little clearer.

Reply:
Oh you're right, shouldn't ever have mixed types

Reply:
Done but note that we can't use toTensorList as that will induce refcount bumps